x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/actions.tsx

```diff
@@ -23,12 +24,31 @@ export const ExplorerLink: FC<Props> = ({ jobsList }) => {
       values: { jobsCount: jobsList.length, jobId: jobsList[0] && jobsList[0].id },
     }
   );
-  const { createLinkWithUserDefaults } = useCreateADLinks();
+  const [explorerPath, setExplorerPath] = useState<string>();
 
-  return (
+  const mlLocator = useMlLocator();
+
+  useEffect(
+    function generateExplorerUrl() {
+      mlLocator!
+        .getUrl({
+          page: ML_PAGES.ANOMALY_EXPLORER,
+          pageState: {
+            jobIds: jobsList.map((job) => job.id),
+          },
+        })
+        .then((path) => {
+          setExplorerPath(path);
+        })
+        .catch((e) => {});
+    },
+    [jobsList]
+  );
+
+  return explorerPath ? (
```

Comment:
Would be great to add an action to the Jobs list here, filtered to show just the selected job group. Either an extra 'Manage job' action, or the table in Stack Management uses a link from the (job) ID.

Reply:
I've refactored actions in 49b1f13a92514bf49c96cee349278eae040b94bf so they work as buttons now